docs/user-guide/deferred-download.rst

```diff
@@ -53,19 +65,87 @@ The packages necessary for deferred downloading can be installed with the follow
 Ensure that the ``httpd``, ``squid``, and ``pulp_streamer`` services are running and enabled
 to start at boot.
 
+.. note::
+
+  If you wish, you can replace ``httpd`` and ``squid`` with the reverse proxy and caching
+  proxy of your choice. However, you will still need the ``python-pulp-streamer``.
+
 
 Configuration
 -------------
 
-In addition to configuring the services listed above, alternate download policies must be
-enabled in the Pulp server configuration. This is located by default in ``/etc/pulp/server.conf``
-and is documented inline. All relevant settings are contained in the ``lazy`` section.
-
-Once the Pulp server is configured, the Apache reverse proxy, Squid, and the Pulp streamer
-require configuration. A default configuration for Apache is provided by the
-``python-pulp-streamer`` package and documentation can be found inline. Squid requires more
-configuration which is up to the user. To configure squid, edit ``/etc/squid/squid.conf``. The
-following is a basic configuration with inline documentation::
+If you have chosen to use the same host as the Pulp server, all the default settings in the
+``lazy`` section of ``/etc/pulp/server.conf`` should work for you. If not, you will need to
+make some configuration adjustments so that Pulp redirects clients to the correct host.
+All configuration options are documented inline. Once the Pulp server is configured, the
+Apache reverse proxy, Squid, and the Pulp streamer require configuration.
+
+Reverse Proxy
+^^^^^^^^^^^^^
+
+A default configuration for Apache is provided by the ``python-pulp-streamer`` package
+and is installed to ``/etc/httpd/conf.d/pulp_streamer.conf``, but it is also reproduced
+below for reference::
```

Comment:
I think there is a way to get Sphinx to automatically inject the file here so that it stays up to date as it changes in the future. Could be worth looking in to.